designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/definition/package.scala

```diff
@@ -19,14 +19,16 @@ package object definition {
 
   import pl.touk.nussknacker.engine.api.CirceUtil._
 
-  @JsonCodec(encodeOnly = true) final case class UIProcessObjects(
-      componentGroups: List[ComponentGroup],
+  // This class contains various views on definitions, used in a different FE contexts
+  @JsonCodec(encodeOnly = true) final case class UIDefinitions(
+      // This is dedicated view for the components toolbox panel
+      componentGroups: List[UIComponentGroup],
       components: Map[ComponentInfo, UIComponentDefinition],
       classes: List[TypingResult],
       // TODO: remove it, use components field on the FE side instead
       componentsConfig: Map[String, SingleComponentConfig],
```

Comment:
I have plan to not return `SingleComponentConfig`, take a look at TODO above. These information will be a part of `UIComponentDefinition` - maybe in some dedicated class, or just inlined in  `UIComponentDefinition`. It will be done in the next PRs. `UITypingResult` would be nice, but it is a little bit bigger thing. We should split into input DTO and output DTO. e.g. in input shouldn't be needed `display`. Maybe we will do it in some future :)